helm/chart/router/values.yaml

```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 # -- See https://www.apollographql.com/docs/router/configuration/overview#configuration-file for yaml structure
 router:
   configuration:
-    supergraph:
+    graphql:
```

Comment:
I thought this should be supergraph.

Reply:
[b2adbaa](https://github.com/apollographql/router/pull/1859/commits/b2adbaa6daf656cb9aa6d4bd3db94755bfa3ffa8)